components/fxa-client/src/internal/device.rs

```diff
@@ -295,7 +303,7 @@ impl FirefoxAccount {
     // because the server does not have a `PATCH commands` endpoint yet.
     #[allow(dead_code)]
     pub(crate) fn unregister_command(&mut self, _: &str) -> Result<()> {
-        self.state.device_capabilities.clear();
+        self.state.clear_last_sent_device_capabilities();
```

Comment:
This kind of goes against Tarik's suggestion of having `StateManager` focus on the low-level state rather than higher-level operations.  If that struct is just managing the state, then I think `clear_last_sent_device_capabilities()` makes perfect sense.  I do like the idea of some sort of "device needs capability registration" function, but I think that should live somewhere else.